patch/kconfig-inclusions

```diff
@@ -49,6 +49,13 @@ CONFIG_I2C_MUX_REG=m
 CONFIG_I2C_MUX_PCA954x=m
 CONFIG_I2C_MUX_MLXCPLD=m
 CONFIG_I2C_MLXCPLD=m
+CONFIG_I2C_DESIGNWARE_PLATFORM=m
+CONFIG_ACPI=y
+CONFIG_IOSF_MBI=y
+CONFIG_ARCH_SUPPORTS_ACPI=y
```

Comment:
Do `CONFIG_ACPI` and `CONFIG_ARCH_SUPPORTS_ACPI` need to be set to `y` here? The former is already enabled due to many other kernel configs that depend on this, and the latter is a feature existence config that depends on the architecture the kernel is being compiled for.

Reply:
Thanks.
we added it into kconfig-inclusions just want to make sure it always be “y”.
Eg. Currently, they are “y” due to dependencies. If in the future the dependencies are removed, they can be “n”, that is something we want to avoid.
But if the dependencies will never be removed, I’m OK to remove them from the kconfig-inclusions.
Let me know how you think.

Reply:
Removed both after offline discussion. Thanks